src/atoms/Tabs/Tabs.stories.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+export default (asStory: *) => {
+  asStory('ATOMS/Tabs', module, (story, { Tabs, Row, Button }) => {
+    story
+      .add('default', () => (
+        <Tabs.Plate defaultSelectedTabId="1">
+          <Row gap="xs">
+            <Tabs.Title tabId="1"><Button>Tab-1</Button></Tabs.Title>
```

Comment:
Can we use tab indexes as id by default?

Reply:
No. I don't like this idea.

Reply:
And we can't do it, because I used context instead of cloneElement.